packages/pages/package.json

```diff
@@ -77,14 +77,14 @@
     "ink": "^3.2.0",
     "lodash": "^4.17.21",
     "mime-types": "^2.1.35",
-    "module-from-string": "^3.2.1",
+    "module-from-string": "^3.3.0",
```

Comment:
I just updated both of these deps to match the lockfile, for some reason when I ran pnpm install it ended up downgrading these (not exactly sure why, maybe I already had these versions in my node_modules)